bad_code_test.py

```
@@ -0,0 +1,60 @@
+# This function violates multiple rules:
+# - No docstring
+# - Uses print() (should be disallowed)
+# - Too long (>50 lines)
+# - Bad naming convention (camelCase)
+def CalculateArea(radius):
+    print("Calculating area...")  # Violates disallow_print_statements
+    if radius < 0:
+        print("Invalid radius!")  # Violates again
+        return None
+    area = math.pi * radius * radius
+    for i in range(60):  # Just to exceed line length rule
```

Ruff (0.14.4)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)